Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+#!groovy
+
+pipeline {
+  agent any
+  // save some io during the build
+  options { durabilityHint('PERFORMANCE_OPTIMIZED') }
+  
+  stages {
+    stage("Parallel Stage") {
+      parallel {
+        stage("Build / Test - mvn latest - JDK8 - ubuntu") {
+          agent { node { label 'ubuntu' } }
+          steps {
+              timeout( time: 180, unit: 'MINUTES' ) {
+                mavenBuild( "jdk_1.8_latest", "maven_latest")
+            }
+          }
+        }
+        stage("Build / Test - mvn latest - JDK11 - ubuntu") {
+          agent { node { label 'ubuntu' } }
+          steps {
+              timeout( time: 180, unit: 'MINUTES' ) {
+                mavenBuild( "jdk_11_latest", "maven_latest")
+            }
+          }
+        }
+        stage("Build / Test - mvn latest - JDK8 - windowx") {
+          agent { node { label 'ubuntu' } }
+          steps {
```

Comment:
? name for windows ... but node for ubuntu

Reply:
already fixed give me 10 minutes when doing something